addons/common/fnc_isAttenuated.sqf

```diff
@@ -0,0 +1,96 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_isAttenuated
+
+Description:
+    Checks whether a unit is attenuated.
+
+Parameters:
+    _unit - Unit to check <OBJECT>
+
+Returns:
+    true - for inside unit that cant hear the outside <BOOL>
+
+Examples:
+    (begin example)
+        if ([player] call CBA_fnc_isAttenuated) then {
+            player sideChat "I cant hear you outside guy!";
+        };
+    (end)
+
+Config & Variable API:
```

Comment:
Will this work with auto-generated API pages?

Reply:
no i dont know auto-gen API pages do you have an example where it is used?

Reply:
Sorry, was a question to @commy2 and @Killswitch00! :)